CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Backtrace Unity Release Notes
 
+## Version 3.4.0-rc1
+- native/managed attachment support - user can add path to attachments via BacktraceClient Initialize method or via Unity Editor Backtrace Window. Once user add them on the initialization method/game object - it will be available in every report that BacktraceClient will send to Backtrace.
```

Comment:
Not to be too pedantic here, but you should change "Unity Editor Backtrace Window" to "Unity Editor in the BacktraceClient's inspector.  In Unity:  Inspector <> Window.

Reply:
thanks, Will make a change.